traffic_router/core/src/test/java/com/comcast/cdn/traffic_control/traffic_router/core/external/SteeringTest.java

```diff
@@ -272,7 +272,7 @@ public void z_itemsMigrateFromSmallerToLargerBucket() throws Exception {
 		Map<String, String> domains = new HashMap<>();
 		Map<String, Integer> weights = new HashMap<>();
 
-		setupSteering(domains, weights, "internal/api/1.3/steering2.json");
+		setupSteering(domains, weights, "api/2.0/steering2");
 
 		List<String> randomPaths = new ArrayList<>();
 
```

Comment:
I can't comment on the file directly, but the `api/2.0/federations` file below probably needs to be `api/2.0/federations/all`

Reply:
the file rename is backwards -- should be `federations/all` not `all/federations`

Reply:
Should be corrected